lib/node_modules/@stdlib/repl/lib/syntax_highlighter.js

```diff
@@ -125,6 +140,138 @@ setNonEnumerableReadOnly( SyntaxHighlighter.prototype, '_highlightLine', functio
 	return highlightedLine;
 });
 
+/**
+* Returns a list of available theme names.
+*
+* @name getThemes
+* @memberof SyntaxHighlighter.prototype
+* @type {Function}
+* @returns {Array} array of all theme names
+*/
+setNonEnumerableReadOnly( SyntaxHighlighter.prototype, 'getThemes', function getThemes() {
+	return objectKeys( this._themes );
+});
+
+/**
+* Sets the current color theme.
+*
+* @name setTheme
+* @memberof SyntaxHighlighter.prototype
+* @type {Function}
+* @param {string} theme - theme name
+* @throws {Error} argument must be an existing theme
+*/
+setNonEnumerableReadOnly( SyntaxHighlighter.prototype, 'setTheme', function setTheme( theme ) {
+	if ( !contains( objectKeys( this._themes ), theme ) ) {
```

Comment:
This is inefficient. Use, e.g., `@stdlib/assert/has-own-property` to check whether the theme exists in the themes object.